pset1.js

```
@@ -30,6 +35,12 @@
     HINT:
         - remember to account for what happens if s > e!
 */
+// const getInRange = (num1, num2) =>{
+//     if(num1 <= num2) newArr.push(num1);
+//     // getInRange(num1+1)
+//     let newArr = getInRange(num1+1)
```

The function would still take in two arguments.

newArr was not declared, so this should throw an error.